enclave-runtime/Enclave.edl

```diff
@@ -88,6 +88,18 @@ enclave {
 			[out, size=unchecked_extrinsic_size] uint8_t* unchecked_extrinsic, uint32_t unchecked_extrinsic_size,
 			int skip_ra
 		);
+		public sgx_status_t generate_dcap_ra_quote(
+			int skip_ra,
+			[in] const sgx_target_info_t* quoting_enclave_target_info,
+			uint32_t quote_size,
+			[out, size=dcap_quote_size] uint8_t* dcap_quote_p, uint32_t dcap_quote_size
+		);
+
+		public sgx_status_t generate_dcap_ra_extrinsic_from_quote_internal(
```

Comment:
Ah sorry, I wasn't very clear here. I think that we should stay consistent for now, and don't use any suffix in the FFI. Maybe we want to introduce this later at some point, consistently for all FIIs.

Reply:
No worries :D